GatewayServer/server.js

```diff
@@ -214,6 +220,11 @@ class ExpressServer {
       }
     });
 
+    router.get("/logout", async(req, res) => {
+      res.clearCookie("jwt");
+      res.send("Logout successfully");
+    });
+
```

Comment:
This is sufficient for our purposes, but note that a user could still note the JWT and use it for further requests. For additional security AuthServer could keep a blacklist of tokens that were logged out. But we def don't need to do this :>